.github/workflows/ci.yaml

```diff
@@ -78,3 +81,17 @@ jobs:
           cosign verify-attestation --key k8s://tekton-chains/signing-secrets "${IMAGE_URL}"
 
           kubectl wait --timeout=5m --for=condition=ready pods -l app=picalc -n prod
+
+ tests:
```

Comment:
The ` tests:` job is not indented correctly, causing the CI to fail.

Reply:
Fixed